src/content/user-guide/arm-alpha.mdx

```diff
@@ -0,0 +1,213 @@
+---
+title: "DeepSparse ARM Alpha"
+metaTitle: "DeepSparse ARM Alpha"
+metaDescription: "Description of how to trial DeepSparse on ARM"
+index: 6000
+---
+
+# DeepSparse ARM Alpha
+
+DeepSparse has Alpha support for ARM platforms! This page explains how to trial it.
+
+[Sign up for early access to the general release of DeepSparse on ARM](https://neuralmagic.com/deepsparse-arm-waitlist)
+
+## Want to Particpate in Our Trial?
+
+Neural Magic is offering a $50 Amazon Gift card for a 30 minute conversation to discuss: 
+- Experience with the ARM Alpha
+- Requests for features for DeepSparse on ARM
+- Requests for ARM platform support (e.g. Raspberry Pi, Rockchip, Qualcomm)
+
+[Reach out to set up time to discuss](https://neuralmagic.com/contact/)
+
+## Understand Alpha Status
+
+DeepSparse on ARM is currently in Alpha state. 
+
+It has the following limitations:
+- **Long Compile Times** (compiling BERT on `M6g.16xlarge` takes XXX minutes)
```

Comment:
don't forget to fill in the xxx minutes (even if it's an approximate range if that was the intent here; it's hard to know if you are going to fill it in)